src/PhpWord/Element/AbstractElement.php

```diff
@@ -108,13 +108,27 @@
     protected $mediaRelation = false;
 
     /**
-     * Is part of collection; true for Title, Footnote, Endnote, and Chart
+     * Is part of collection; true for Title, Footnote, Endnote, Chart, and Comment
      *
      * @var bool
      */
     protected $collectionRelation = false;
 
     /**
+     * The start position for the linked comment
+     * 
+     * @var Comment
+     */
+    protected $commentStart;
```

Comment:
Should this be part of `AbstractElement`? Seems it should be part of `Comment` to me. Or is it because comments should be attached to an element? If so, maybe the attribute name could be clarified just a bit more? 👍